### PR TITLE
Add build script for feather-format

### PR DESCRIPTION
**Pull request by ogrisel**
https://github.com/wesm/feather

### CODE REVIEW

File: build_feathers.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#    docker run --rm -e FEATHER_VERSIONS=0.17.0 -e PYTHON_VERSIONS=2.7 -v $PWD:/io quay.io/pypa/manylinux1_x86_64 /io/build_feathers.sh
+#
+# Make sure numpy and Cython wheels are on the manylinux server or built in the
+# $WHEELHOUSE directory first.
+set -e
+
+# Manylinux, Python versions
+if [ -z "${PYTHON_VERSIONS}" ]; then
+    PYTHON_VERSIONS="2.7 3.4 3.5"
+fi
```
Comment (ogrisel) [inline build_feathers.sh]: I am not sure about that. Should I instead skip PYTHON < 2.7 for Python 2 and PYTHON  < 3.4 for Python  3 in the main build loop?

Comment (matthew-brett) [inline build_feathers.sh]: Could do - I use this list for when I'm building lots of versions, for upload to the nipy server - or if projects will accept them - like numpy.  But you could do both - skip versions you know cannot build in the main loop.

Comment (ogrisel) [inline build_feathers.sh]: Actually I haven't tried but feather only supports officially Python 2.7 and 3.4+ so I think we should not build for versions of Python that are not officially supported and tested by upstream, even when it's possible to generate the wheel file.

Comment (ogrisel) [inline build_feathers.sh]: Let me amend my commit to skip inside the for loop instead.

**Comment by matthew-brett** [comment]
For numpy etc, I used numpy version checks in the main loop to abort attempts to build numpy for incompatible Pythons.  Of course this is mainly useful when you are thinking of compiling older versions of a package.

**Comment by ogrisel** [comment]
I amended the comment to skip unsupported versions in the main loop in a similar way to what is done for the numpies. travis is green. circileci is still red but thus AFAIK this is unrelated to this PR.

**Comment by matthew-brett** [comment]
Nice - thanks a lot.

**Pull request merged**